Permanent/ViewControllers/Profile/PublicProfileMilestonesViewController.swift

```diff
@@ -98,7 +99,44 @@ extension PublicProfileMilestonesViewController: UITableViewDataSource, UITableV
         return cell
     }
     
-    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        tableView.deselectRow(at: indexPath, animated: true)
+    func tableView(_ tableView: UITableView, trailingSwipeActionsConfigurationForRowAt indexPath: IndexPath) -> UISwipeActionsConfiguration? {
+        let item = viewModel?.milestonesProfileItems[indexPath.row]
+        
+        let deleteAction = UIContextualAction.make(
+            withImage: .deleteActionRed,
+            backgroundColor: .destructive,
+            handler: { _, _, completion in
+                self.showSpinner()
```

Comment:
you should do weak self here and below. This is a retain cycle where the viewcontroller holds onto the tableview and the tableview holds on to the viewcontroller 

Please use weak self in the handler block below as well